cli/flox/src/commands/list.rs

```diff
@@ -53,9 +55,16 @@ pub enum ListMode {
     All,
 }
 
+impl Display for ListMode {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
+
 impl List {
-    #[instrument(name = "list", fields(mode), skip_all)]
+    #[instrument(name = "list", skip_all)]
     pub async fn handle(self, flox: Flox) -> Result<()> {
+        set_tag("list_mode", &self.list_mode);
```

Comment:
this could also just be `&format!("{:?}", self.list_mode)` at least we wouldnt need to implement Display == Debug

Reply:
Do you have a preference?

Reply:
slightly prefer being explicit there over a special display impl

alternatively we can impl display or some `-> String` method /function that produces a slightly less debuggy string for easier querying, e.g. just comma delimited

Reply:
Done (format) in rebase